Gemfile

```diff
@@ -3,15 +3,16 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.5.1'
 
-gem 'rails', '~> 6.1.1'
+gem 'rails', '~> 6.0'
 gem 'pg', '~> 1.1'
 gem 'puma', '~> 5.0'
 gem 'bootsnap', '>= 1.4.4', require: false
 gem 'rack-cors', require: 'rack/cors'
+gem 'devise_token_auth'
 
 group :development, :test do
   gem 'pry-rails'
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'pry-byebug'
```

Comment:
why have you added another pry gem?

Reply:
we had initial problems with the set up and changed this gem to pry-byebug after looking at an old API repo. change it back?